src/hooks/useKanban.js

```diff
@@ -233,7 +241,18 @@ visitorId = 'visitor_' + Date.now() + '_' + Math.random().toString(36).substr(2,
       const sourceColumn = columns[sourceColumnId];
       const destinationColumn = columns[destinationColumnId];
       
-      if (!sourceColumn || !destinationColumn) return;
+      // Se alguma das colunas não existir, espera um pouco e tenta novamente
+      if (!sourceColumn || !destinationColumn) {
+        console.log('Aguardando colunas carregarem...');
+        await new Promise(resolve => setTimeout(resolve, 500));
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
const retryOperation = async (operation, maxAttempts = 3, initialDelay = 500) => {
  for (let i = 0; i < maxAttempts; i++) {
    const result = await operation();
    if (result) return result;
    await new Promise(resolve => setTimeout(resolve, initialDelay * Math.pow(2, i)));
  }
  return null;
};
```
The retry mechanism uses a hardcoded timeout value and only retries once. Implement a more robust retry mechanism with configurable attempts and exponential backoff.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;